content/_data/roadmap/roadmap.yml

```
@@ -54,10 +59,33 @@ categories:
     - name: "Pipeline: GitHub App authentication"
       status: current
       link: https://issues.jenkins-ci.org/browse/JENKINS-57351
+    - name: "Git Plugin Performance Improvements"
+      status: current
+      description: >
+        Improve Jenkins git plugin performance by fixing known issues in performance critical areas
+      link: https://www.jenkins.io/projects/gsoc/2020/projects/git-plugin-performance/
     - name: "GitHub Checks API integrations"
-      status: near-term
-      link: https://jenkins.io/projects/gsoc/2020/project-ideas/github-checks/
-    - name: Pipeline as YAML
+      status: current
+      description: >
+        Create a new plugin API so that plugins can publish GitHub Checks statuses.
+        Implement Checks API support in Warnings NG and Code Coverage API plugins.
+      link: https://www.jenkins.io/projects/gsoc/2020/projects/github-checks/
+    - name: Machine Learning Plugin for Data Science
+      status: current
+      description: >
+        Integrating Machine Learning workflow with Jenkins build tasks, including Data preprocessing, Model Training, Evaluation and Prediction.
```

Other roadmap descriptions are using the more direct verb (for example, create, expose, enhance).  Might consider more direct verb here rather than using a gerund.
```suggestion
        Integrate Machine Learning workflow with Jenkins build tasks, including Data preprocessing, Model Training, Evaluation and Prediction.
```